packages/perseus/src/jipt-paragraphs.js

```diff
@@ -11,28 +11,36 @@ const arrayRules: ParserRules = {
     fence: {
         match: SimpleMarkdown.defaultRules.fence.match,
         order: 1,
-        // $FlowFixMe[incompatible-type]
-        parse: (capture, state, parse) => capture[3],
+        parse: (capture, state, parse) => ({
+            type: "codeBlock",
+            lang: capture[2] || undefined,
+            content: capture[3],
+        }),
     },
     paragraph: {
         match: SimpleMarkdown.defaultRules.paragraph.match,
         order: 2,
-        // $FlowFixMe[incompatible-type]
-        parse: (capture, state, parse) => capture[1],
+        parse: (capture, state, parse) => ({
+            content: capture[1],
+        }),
     },
 };
 
-// $FlowFixMe[prop-missing]
-const builtArrayParser = SimpleMarkdown.parserFor(arrayRules);
+const builtArrayParser: $Call<
+    typeof SimpleMarkdown.parserFor,
+    typeof arrayRules,
+> = SimpleMarkdown.parserFor(arrayRules);
 
 // This should just return an array of strings! magick!
-const parseToArray = (source: string): $FlowFixMe => {
+const parseToArray = (source: string): Array<string> => {
```

Comment:
In the original iteration of this PR I was returning the wrong value and type.  I thought it was a direct return of whatever the built `simple-markdown` parser returns. But all the callers (and the comment directly above this function) want it to be an array of strings (the parsed content).  

I've fixed this now and it should result in no changes needed in webapp!